retry/src/main/java/io/micronaut/retry/intercept/CircuitBreakerRetry.java

```diff
@@ -93,7 +93,7 @@ public void close(@Nullable Throwable exception) {
     public void open() {
         if (currentState() == CircuitState.OPEN && lastError != null) {
             if (LOG.isDebugEnabled()) {
-                LOG.debug("Rethrowing existing exception for Open Circuit [{}]: {}", method, lastError.getMessage());
+                LOG.debug("Rethrowing existing exception for Open Circuit [{}]: {}", method, lastError.getMessage(), lastError);
```

Comment:
Rethrow again likely does not require a stacktrace at this point.
```suggestion
                LOG.debug("Rethrowing existing exception for Open Circuit [{}]: {}", method, lastError.getMessage());
```